p4-16/psa/PSA.mdk

```diff
@@ -835,6 +853,12 @@ For Ethernet ports, any padding required to get the packet up to the
 minimum frame size required is done by the implementation, as well as
 calculation of and appending the Ethernet frame CRC.
 
+It is expected that typical P4 programs will have explicit checks to
+avoid sending packets larger than a port's maximum frame size.  A
+typical implementation will drop frames larger than this maximum
+supported size.  It is recommended that they maintain error counters
+for such dropped frames.
```

Comment:
Again, seems obvious that something like this must be the case, but comment if you disagree.